functions/go/create-setters/README.md

```diff
@@ -4,8 +4,8 @@
 
 <!--mdtogo:Short-->
 
-Add setter comments to matching resource fields. Setters serve as parameters 
-for template-free setting of comments.
+Add setter comments to matching resource fields. Setters serve as
+parameters for the template-free setting of field values.
```

Comment:
This overview is great if one is already familiar with `setters`, but users who aren't, its a good idea to spend 2-3 sentences explaining the problem with an example. Like in this case, taking about, the use-case for parameterization (for ex. being able to change replicas of the deployment, or change the version of the underlying application). That's sets the narrative for parameterization, and then following up with how function let you achieve that.